co-log-core/src/Colog/Core/IO.hs

```diff
@@ -150,3 +151,11 @@ foo
 liftLogIO :: MonadIO m => LogAction IO msg -> LogAction m msg
 liftLogIO (LogAction action) = LogAction (liftIO . action)
 {-# INLINE liftLogIO #-}
+
+{- | This action can be used in combination with other actions to flush
+   a handle every time you log anything.
+-}
```

Comment:
That makes sense, but should I write literally `x.x.x.x` or some specific version? The problem with the latter is that I don't know what version will be released next, it can be `0.2.2` or `0.2.1.2` or something else, probably that's why you suggest `x.x.x.x`. So I just want to clarify: do you update `@since x.x.x.x` to `@since a.b.c.d` when you make a new release `a.b.c.d`?

Reply:
I meant literally `@since x.x.x.x`. We haven't planned the milestone for the next version, so at this point it's unknown what is the exact next version. But before the release, it's much easier to grep for `@since x.x.x.x` and update those annotations rather then browsing the whole commit history from the latest release.